File: code/web3.py
$ ipython
In [1]: from web3 import Web3, HTTPProvider
In [2]: import os
In [3]: INFURA_API = os.environ['INFURA_API']
In [4]: w3 = Web3(HTTPProvider('https://ropsten.infura.io/'+INFURA_API))
In [5]: w3.eth.blockNumber
Out[5]: 2872088

File: code/web3.js
node
    > Web3 = require('web3');
> INFURA_API = process.env.INFURA_API; // Infura is a third party service that allows us to connect to their Ethereum node without setting up our own. > web3 = new Web3(new Web3.providers.HttpProvider("https://mainnet.infura.io/" + INFURA_API));
> web3.eth.blockNumber;
5289236


